example/build_cmake_installed/CMakeLists.txt

```diff
@@ -10,7 +10,9 @@ project(Example CXX Fortran)
 find_package(Kokkos REQUIRED)
 
 add_executable(example cmake_example.cpp foo.f)
-set_target_properties(example PROPERTIES LINKER_LANGUAGE Fortran)
+if(CMAKE_Fortran_COMPILER_ID STREQUAL LLVMFlang)
```

Comment:
Only LLVMFlang, not also Flang?
Also for the record LLVMFlang was added in CMake 3.24
https://cmake.org/cmake/help/v3.24/release/3.24.html#compilers
https://cmake.org/cmake/help/v3.23/variable/CMAKE_LANG_COMPILER_ID.html does not mention it
(not saying it is an issue)

Reply:
Yeah, but for older versions flang usage is broken anyway, so it won't matter much.